src/gui/mainwindow.cpp

```diff
@@ -60,3 +74,37 @@ MainWindow::~MainWindow() {
 void MainWindow::openGitHubLink(void) {
     QDesktopServices::openUrl(QUrl("https://github.com/linescaleGUI/linescaleGUI"));
```

Comment:
Use `version::repoUrl` from `version.h`.

Reply:
Prefer to leave the main repo link as is; converted gitHash in the line above to a link to the commit <br>
![image](https://user-images.githubusercontent.com/91913978/206809565-18cd7b99-1ed6-4bd6-a876-2df31f0dae43.png)